variants/tracker-t1000-e/variant.h

```diff
@@ -0,0 +1,158 @@
+
+/** Master clock frequency */
+#define VARIANT_MCK (64000000ul)
+
+#define USE_LFXO // Board uses 32khz crystal for LF
+// define USE_LFRC    // Board uses RC for LF
+
+/*----------------------------------------------------------------------------
+ *        Headers
+ *----------------------------------------------------------------------------*/
+
+#include "WVariant.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+
+// Number of pins defined in PinDescription array
+#define PINS_COUNT (48)
+#define NUM_DIGITAL_PINS (48)
+#define NUM_ANALOG_INPUTS (6)
+#define NUM_ANALOG_OUTPUTS (0)
+
+#define PIN_3V3_EN (32 + 6) // P1.6, Power to Sensors
+#define PIN_3V3_ACC_EN (32 + 7) // P1.7, Power to Acc
+
+#define PIN_LED1 (0 + 24) // P0.24
+#define LED_PIN PIN_LED1
+#define LED_BUILTIN -1
+#define LED_BLUE -1 // Actually green
+#define LED_STATE_ON 1 // State when LED is lit
+
+#define BUTTON_PIN (0 + 6) // P0.6
+#define BUTTON_ACTIVE_LOW false
+#define BUTTON_ACTIVE_PULLUP false
+#define BUTTON_SENSE_TYPE INPUT_SENSE_HIGH
+
+#define HAS_WIRE 1
+
+#define WIRE_INTERFACES_COUNT 1
+
+#define PIN_WIRE_SDA (0 + 9) // P0.26
+#define PIN_WIRE_SCL (0 + 10) // P0.27
+
+/*
+ * Serial interfaces
+ */
+#define PIN_SERIAL1_RX (0 + 14) // P0.14
+#define PIN_SERIAL1_TX (0 + 13) // P0.13
+
+#define PIN_SERIAL2_RX (0 + 17) // P0.17
+#define PIN_SERIAL2_TX (0 + 16) // P0.16
+
+
+#define SPI_INTERFACES_COUNT 1
+
+#define PIN_SPI_MISO (32 + 8) // P1.08
+#define PIN_SPI_MOSI (32 + 9) // P1.09
+#define PIN_SPI_SCK (0 + 11)  // P0.11
+#define PIN_SPI_NSS (0 + 12)  // P0.12
+
+#define LORA_RESET (32 + 10) // P1.10 // RST
+#define LORA_DIO1 (32 + 1)   // P1.01 // IRQ
+#define LORA_DIO2 (0 + 7)    // P0.07 // BUSY
+#define LORA_SCK PIN_SPI_SCK
+#define LORA_MISO PIN_SPI_MISO
+#define LORA_MOSI PIN_SPI_MOSI
+#define LORA_CS PIN_SPI_NSS
+
+// supported modules list
+#define USE_LR1110
+
+#define LR1110_IRQ_PIN LORA_DIO1
+#define LR1110_NRESER_PIN LORA_RESET
+#define LR1110_BUSY_PIN LORA_DIO2
+#define LR1110_SPI_NSS_PIN LORA_CS
+#define LR1110_SPI_SCK_PIN LORA_SCK
+#define LR1110_SPI_MOSI_PIN LORA_MOSI
+#define LR1110_SPI_MISO_PIN LORA_MISO
+
+#define LR11X0_DIO3_TCXO_VOLTAGE 1.6
+#define LR11X0_DIO_AS_RF_SWITCH
+#define LR11X0_DIO_RF_SWITCH_CONFIG 0x0f, 0x0, 0x09, 0x0B, 0x0A, 0x0, 0x4, 0x0
+
+#define HAS_GPS 1
+#define GNSS_Airoha
+#define GPS_RX_PIN PIN_SERIAL1_RX
+#define GPS_TX_PIN PIN_SERIAL1_TX
+
+#define GPS_BAUDRATE 115200
+
+#define PIN_GPS_EN (32 + 11) // P1.11
+#define GPS_EN_ACTIVE HIGH
+
+#define PIN_GPS_RESET (32 + 15) // P1.15
+#define GPS_RESET_MODE HIGH
+
+#define GPS_VRTC_EN (0 + 8) // P0.8, awlays high
+#define GPS_SLEEP_INT (32 + 12) // P1.12, awlays high
+#define GPS_RTC_INT (0 + 15) // P0.15, normal is LOW, wake by HIGH
+#define GPS_RESETB_OUT (32 + 14) // P1.14, awlays input pull_up
+
+// #define GPS_THREAD_INTERVAL 50
+#define GPS_FIX_HOLD_TIME 15000 // ms
+
+#define BATTERY_PIN 2
+// #define ADC_CHANNEL ADC1_GPIO2_CHANNEL
```

Comment:
consider removing comment before submitting patch

Reply:
done for that